{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5963", 
                "published": "2015-08-24T10:59:08.773-04:00", 
                "id": "CVE-2015-5963", 
                "modified": "2015-08-25T13:40:58.593-04:00", 
                "summary": "contrib.sessions.middleware.SessionMiddleware in Django 1.8.x before 1.8.4, 1.7.x before 1.7.10, 1.4.x before 1.4.22, and possibly other versions allows remote attackers to cause a denial of service (session store consumption or session record removal) via a large number of requests to contrib.auth.views.logout, which triggers the creation of an empty session record."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/399.html", 
                "id": "CWE-399", 
                "title": "Resource Management Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:djangoproject:django:1.8.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.8.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.6"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.8.0"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.6"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.9"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.7"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.8.3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.10"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.7"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.11"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.8"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.8"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.14"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7.9"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:rc2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.19"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.12"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.13"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:rc3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:rc1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.17"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:12.04::~~lts~~~"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.8:beta1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.21"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:14.04::~~lts~~~"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.20"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:15.04"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.77329", 
                "id": 77329
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "https://www.djangoproject.com/weblog/2015/aug/18/security-releases/", 
                    "vendor": "MISC"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/USN-2720-1", 
                    "vendor": "UBUNTU"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "2.9", 
                    "access complexity": "low", 
                    "confidentiality impact": "none", 
                    "authentication": "none", 
                    "exploit": "10.0", 
                    "base": "5.0", 
                    "integrity impact": "none", 
                    "availability impact": "partial", 
                    "access vector": "network"
                }
            ], 
            "severity": "Moderate", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2015-5963"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.cisecurity:def:195", 
                "title": "DSA-3338-1 -- python-django -- security update", 
                "class": "patch", 
                "id": "oval:org.cisecurity:def:195"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Debian Security Advisory DSA 3338-1 (python-django - security update)", 
                "file": "deb_3338.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "703338"
            }
        ], 
        "Nessus": [
            {
                "name": "Debian DLA-301-1 : python-django security update", 
                "file": "debian_DLA-301.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "85656"
            }, 
            {
                "name": "Debian DSA-3338-1 : python-django - security update", 
                "file": "debian_DSA-3338.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "85587"
            }, 
            {
                "name": "Fedora 23 : python-django-1.8.6-1.fc23 (2015-1dd5bc998f)", 
                "file": "fedora_2015-1dd5bc998f.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "89168"
            }, 
            {
                "name": "FreeBSD : django -- multiple vulnerabilities (b0e54dc1-45d2-11e5-adde-14dae9d210b8)", 
                "file": "freebsd_pkg_b0e54dc145d211e5adde14dae9d210b8.nasl", 
                "family": "FreeBSD Local Security Checks", 
                "id": "85522"
            }, 
            {
                "name": "openSUSE Security Update : python-Django (openSUSE-2015-592)", 
                "file": "openSUSE-2015-592.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "86047"
            }, 
            {
                "name": "openSUSE Security Update : python-django (openSUSE-2015-598)", 
                "file": "openSUSE-2015-598.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "86091"
            }, 
            {
                "name": "Ubuntu 12.04 LTS / 14.04 / 15.04 : python-django vulnerability (USN-2720-1)", 
                "file": "ubuntu_USN-2720-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "85534"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [], 
        "Redhat": [], 
        "Gentoo": [], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [
            {
                "url": "http://www.ubuntu.com/usn/USN-2720-1", 
                "id": "USN-2720-1"
            }
        ], 
        "Debian": [], 
        "Vmware": []
    }
}